examples/blob-storage/main.tf

```diff
@@ -17,7 +17,7 @@ resource "azuread_service_principal_password" "humanitec_provisioner" {
 
 resource "azurerm_role_assignment" "resource_group" {
   scope                = data.azurerm_resource_group.main.id
-  role_definition_name = "Contributor"
+  role_definition_name = "Owner"
```

Comment:
This is required as this example creates a new role for the workload itself, which requires the [Owner](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-steps#privileged-administrator-roles) role.